sopel/cli/config.py

```diff
@@ -66,6 +70,7 @@ def handle_list(options):
 
     :param options: parsed arguments
     :type options: ``argparse.Namespace``
+    :return: Return 0 if everything went fine.
```

Comment:
```suggestion
    :return: 0 if everything went fine
```

This is our style elsewhere, in existing Sphinx docs.

Also, it's weird that no other return value (exit code) is possible.